core/src/components.d.ts

```diff
@@ -2921,6 +2921,9 @@ export namespace Components {
          */
         "mode"?: "ios" | "md";
     }
+    /**
+     * @experimental The label text to associate with the textarea. Use the `labelPlacement` property to control where the label is placed relative to the textarea. Use this if you need to render a label with custom HTML.
```

Comment:
Why is this experimental tag showing up on top of textarea but it doesn't mention the actual `label` slot or property?

Reply:
Not sure. Looks like this is also happening for the input slot too. Let me check with the Stencil team to see if I'm doing something wrong.

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/27647/commits/de54350bb6540b246df3207bb16422461dc59004